grafana_monitoring/compose.yaml

```diff
@@ -0,0 +1,24 @@
+services:
+  grafana:
+    image: grafana/grafana-oss
+    container_name: grafana-instance
+    environment:
+      - GF_AUTH_GENERIC_OAUTH_CLIENT_ID=$IRIS_CLIENT_ID
+      - GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET=$IRIS_CLIENT_SECRET
+      - GF_AUTH_GENERIC_OAUTH_AUTH_URL=https://iris-iam.stfc.ac.uk/authorize
+      - GF_AUTH_GENERIC_OAUTH_API_URL=https://iris-iam.stfc.ac.uk/userinfo
+      - GF_AUTH_GENERIC_OAUTH_ENABLED=TRUE
+      - GF_AUTH_GENERIC_OAUTH_SCOPES=openid profile email
+      - GF_AUTH_GENERIC_OAUTH_TOKEN_URL=https://iris-iam.stfc.ac.uk/token 
+      - GF_AUTH_GENERIC_OAUTH_GROUPS_ATTRIBUTE_PATH=groups[*]
+      - GF_AUTH_GENERIC_OAUTH_ALLOWED_GROUPS=stfc-cloud/admins
+      - GF_AUTH_GENERIC_OAUTH_DOMAIN=$GRAFANA_DOMAIN_NAME
+      - GF_AUTH_GENERIC_OAUTH_ROOT_URL=$GRAFANA_ROOT_URL
+      - GF_AUTH_GENERIC_OAUTH_PROTOCOL=http
+      - GF_AUTH_GENERIC_OAUTH_HTTP_PORT=3000
+      - PROD_INFLUX_USERNAME=$PROD_INFLUX_USERNAME
+      - PROD_INFLUX_PASSWORD=$PROD_INFLUX_PASSWORD
+      - PROD_OPENSEARCH_PASSWORD=$PROD_OPENSEARCH_PASSWORD
```

Comment:
You will need to add the new variables you created in response to another comment here so that they are passed into the container.